spec.html

```diff
@@ -33287,7 +33289,9 @@
             1. If _s_ is *undefined*, return CreateIterResultObject(*undefined*, *true*).
             1. Assert: _s_ has a [[SetData]] internal slot.
             1. Let _entries_ be the List that is _s_.[[SetData]].
-            1. Repeat, while _index_ is less than the total number of elements of _entries_. The number of elements must be redetermined each time this method is evaluated.
+            1. Let _numEntries_ be the number of elements of _entries_.
```

Comment:
> I think I'd prefer to have an explicit repeat of `let _numEntries_ be the number of elements of _entries_.` as the last loop item.

You mean as the last step in the body of the loop? But then that would re-determine it for each iteration, rather than merely for each evaluation of the method, which is what the note says.

> This seems like a normative requirement encoded in a note.

It seemed to me that the note was just emphasizing something that you'd have to do anyway even if the note weren't there.